apps/proposer/src/app.mjs

```diff
@@ -38,26 +38,13 @@ const nf3 = new Nf3(SIGNING_KEY, environment);
 
 app.set('nf3', nf3);
 
-app.use((req, res, next) => {
-  res.header('Access-Control-Allow-Origin', '*');
-  next();
+setupHttpDefaults(app, app => {
+  app.use('/proposer', proposer);
+  app.use('/contract-address', contracts);
```

Comment:
Made me realise this is different in `challenger` app, where we don't seem to have any routes? :thinking:

Reply:
Yes, it might sound strange, but the `challenger` doesn't have endpoints because it "listens" for Websocket events related to challenge things.